### DISCUSSION TITLE
Accessing another Translatable Field

### DISCUSSION THREAD

**Discussion by DavidZudu** [flyntwp/flynt#402] [Q&A] [answered]
This might be something super simple that I'm missing. 

Is it possible to get the fields of another translatable/option field?

1. Component "Foo" has Translatable fields "X" and "Y" defined.
2. Component "Bar" want to display the content of "X". 

This seems super simple but I'm not sure how to get the field.

**Comment by aaronmeder** [comment, marked as answer]
Hey @DavidZudu 

yes, you can do this easily:

inside your components functions.php file:

```php
add_filter('Flynt/addComponentData?name=Bar', function ($data) {
    $data['trans'] = Options::getTranslatable('Foo');
    return $data;
});
```

Then inside your twig file you can use `{{ trans.X }}` to access the translation. If you cannot access the value try to use {{ dump(trans) }} to see if the data is nested or similar. 

Hint: you need to have `define( 'WP_DEBUG', true );` in your site's wp-config.php to see the dump values.

Hope that helps.
Aaron
